portfolio.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width">
+	<link href="porfolio_normalize.css" rel="stylesheet" type="text/css">
+	<link href="css_portfolio.css" rel="stylesheet" type="text/css">
+</head>
+<body>
+	<div class="navbar">
```

Comment:
Could using a section tag here make the code more semantic. Same applies to the 'about' div on line 28